streamdeck-plugin/src/event_handlers/emoji_react_event_handler.py

```diff
@@ -0,0 +1,37 @@
+from event_handlers.base_event_handler import EventHandler
+
+
+class EmojiReactEventHandler(EventHandler):
+    """
+    A reusable handler for Emoji reactions.
+    """
+
+    ACTION_TO_EMOJI = {
+        "com.chrisregado.googlemeet.emojireact.sparklingheart": "💖",
+        "com.chrisregado.googlemeet.emojireact.thumbsup": "👍",
+        "com.chrisregado.googlemeet.emojireact.partypopper": "🎉",
+        "com.chrisregado.googlemeet.emojireact.clappinghands": "👏",
+        "com.chrisregado.googlemeet.emojireact.facewithtearsofjoy": "😂",
+        "com.chrisregado.googlemeet.emojireact.facewithopenmouth": "😮",
+        "com.chrisregado.googlemeet.emojireact.cryingface": "😢",
+        "com.chrisregado.googlemeet.emojireact.thinkingface": "🤔",
+        "com.chrisregado.googlemeet.emojireact.thumbsdown": "👎"
+    }
+
+    STREAM_DECK_ACTION_PREFIX = "com.chrisregado.googlemeet.emojireact."
+
+    def _get_emoji_char_for_event(self, event: dict):
+        action = event['action']
+        emoji_char = self.ACTION_TO_EMOJI.get(action)
+        if not emoji_char:
+            raise NotImplementedError(f"The action '{action}' was requested, but there is no corresponding emoji.")
+        return emoji_char
+
+    @staticmethod
+    def _make_emoji_react_browser_plugin_message(self, emoji_char):
+        return self._make_simple_sd_event(f"emojiReact={emoji_char}")
```

Comment:
Events between the plugin and browser extension are just arbitrary JSON, so instead of trying to encode the emoji in a single field, you could add another field to the JSON object:

```
return json.dumps({"event": "emojiReact", "emojiChar": emoji_char})
```

And then in [emoji_reaction_event_handler.js](https://github.com/ChrisRegado/streamdeck-googlemeet/pull/33/files#diff-dbc72aea38e576a7395ef39c242d27a085e5025ef0f923818fb964d10928bc43), instead of using a regex, you would have something a bit simpler:
```
  handleStreamDeckEvent = (message) => {
    if (message.event === "emojiReact") {
      this._emojiReact(message.emojiChar);
    }
  }
```

Reply:
Good suggestion, thanks - I hadn't yet spent the time to understand the browser plugin message format.